### ISSUE TITLE
Is the benchmark db size accurate?

### ISSUE THREAD

**Issue by derekperkins** [schollz/goagrep#11]
According to the readme and common sense, the smaller subsets seem like they would be larger file sizes, but the benchmark shows them getting smaller.

![image](https://cloud.githubusercontent.com/assets/3588778/17072292/589c49ce-5024-11e6-9963-3c7b301f4545.png)

**Comment by schollz** [comment]
I think this is something with BoltDB actually, at the time I ran it. BoltDB has some special magic to keep the database larger than needed to make things faster ([more eloquently said here](https://github.com/boltdb/bolt/issues/564)).

That being said, I just re-ran these and they are all 64MB. So that means that they are likely using between 32 MB and 64MB (if I understand BoltDB correctly).

Thanks for the notice though, I will update this. I also noticed that the example doesn't run since I added the new feature so I will fix that too.
